qiskit_experiments/calibration/calibrations.py

```diff
@@ -0,0 +1,657 @@
+        """
+        if schedule is None:
+            schedule = self.get_schedule(schedule_name, qubits, free_params, group)
+
+        gate = Gate(name=schedule_name, num_qubits=len(qubits), params=list(schedule.parameters))
+        circ = QuantumCircuit(len(qubits), len(qubits))
+        circ.append(gate, list(range(len(qubits))))
+        circ.add_calibration(gate, qubits, schedule, params=schedule.parameters)
+
+        return circ
+
+    def schedules(self) -> List[Dict[str, Any]]:
+        """
+        Return the schedules in self in a list of dictionaries to help
+        users manage their schedules.
+
+        Returns:
+            data: A list of dictionaries with all the schedules in it.
+        """
+        data = []
+        for context, sched in self._schedules.items():
+            data.append({"context": context, "schedule": sched, "parameters": sched.parameters})
+
+        return data
+
+    def parameters_table(
+        self,
+        parameters: List[str] = None,
+        schedules: List[Union[Schedule, str]] = None,
+        qubit_list: List[Tuple[int, ...]] = None,
+    ) -> List[Dict[str, Any]]:
+        """
+
+        Args:
+            parameters: The parameter names that should be included in the returned
+                table. If None is given then all names are included.
+            schedules: The schedules to which to restrict the output.
+            qubit_list: The qubits that should be included in the returned table.
+                If None is given then all channels are returned.
+
+        Returns:
+            data: A dictionary of parameter values which can easily be converted to a
+                data frame.
+        """
+
+        data = []
+
+        # Convert inputs to lists of strings
+        if parameters is not None:
+            parameters = {prm.name if isinstance(prm, Parameter) else prm for prm in parameters}
+
+        if schedules is not None:
+            schedules = {sdl.name if isinstance(sdl, Schedule) else sdl for sdl in schedules}
+
+        # Look for exact matches. Default values will be ignored.
+        keys = set()
+        for key in self._params.keys():
+            if parameters and key.parameter not in parameters:
+                continue
+            if schedules and key.schedule not in schedules:
+                continue
+            if qubit_list and key.qubits not in qubit_list:
+                continue
+
+            keys.add(key)
+
+        for key in keys:
+            for value in self._params[key]:
+                value_dict = dataclasses.asdict(value)
+                value_dict["qubits"] = key.qubits
+                value_dict["parameter"] = key.parameter
+                value_dict["schedule"] = key.schedule
+
+                data.append(value_dict)
+
+        return data
+
+    def to_csv(self):
+        """
+        Serializes the parameterized schedules and parameter values so
+        that they can be stored in csv file.
```

Comment:
I improved the docstring here: afaba9b. This is the minimal example that we need to save what is going on in the calibrations. Later it would be good to improve the functionality of saving and loading cals.